### PR TITLE
Add 4noobs

### PR DESCRIPTION
**Pull request by edersonferreira**
Add [4noobs](https://github.com/he4rt/4noobs) to the list of repositories

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,3 +122,5 @@ See all meetups and events [here](https://hacktoberfest.digitalocean.com/events)
 [![CC0](http://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg)](http://creativecommons.org/publicdomain/zero/1.0)
 
 To the extent possible under law, [Otacilio Maia](https://github.com/OtacilioN) has waived all copyright and related or neighbouring rights to this work.
+
+
```
Comment (OtacilioN) [inline README.md:126]: Can you remove this duplicated empty line at the end?

**Review by OtacilioN — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (edersonferreira) [reply README.md]: Yes. now I removed it.

Comment (edersonferreira) [reply README.md]: Yes. now I removed it.

**Review by OtacilioN — APPROVED** [review]
Awesome, thanks for contributing!

**Pull request merged**